src/scss/cdb-components/shapes/threePoints.scss

```diff
@@ -71,6 +106,9 @@ Description
 
 .CDB-Shape-threePoints {
   display: inline-block;
+  &.is-horizontal {
```

Comment:
As we said in the styleguide, we should try to avoid nesting, so this should be rewritten:

```.CDB-Shape-threePoints.is-horizontal { /* … */ }```

Ditto for ```.is-round```, etc. in the next lines